backend/grants/models.py

```diff
@@ -110,7 +110,6 @@ class ApprovedType(models.TextChoices):
     age_group = models.CharField(
         _("Age group"), max_length=20, choices=AgeGroup.choices, blank=True
     )
-    age = models.PositiveSmallIntegerField(_("age"), null=True)  # TODO: remove
```

Comment:
do we want to remove this field? I guess we won't need the legacy data

Reply:
yes, I want to remove this since now we have AgeGroup